src/e2e-test/features/bigquery/source/BigQueryToBigQuery.feature

```diff
@@ -262,3 +262,81 @@ Feature: BigQuery source - Verification of BigQuery to BigQuery successful data
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Validate the values of records transferred to BQ sink is equal to the values from source BigQuery table
+
+  @BQ_EXISTING_SOURCE_TEST @BQ_EXISTING_SINK_TEST
+  Scenario: Validate successful record transfer from BigQuery source(existing table) to BigQuery sink(existing table)
```

Comment:
for this scenario use connection is true for both source and sink in ITN , please use .

Reply:
resolved